Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -204,7 +323,6 @@ public function method444f1()
     }
 
     /**/
-
```

Comment:
> @GrahamCampbell this is the "expected" part

Exactly. The expectation hasn't changed for the output. This fixer is broken if it removes that space.